### PR TITLE
Provide documentation around injecting Aura.Router

### PR DESCRIPTION
**Pull request by weierophinney**
Provides documentation around creating your own Aura.Router instance, configuring it, and injecting it into your router instance for zend-expressive. Both programmatic and IoC approaches are detailed.

### CODE REVIEW

File: doc/book/router.aura.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+use that when creating your `Application` instance.
+
+```php
+<?php
+use Aura\Router\RouterFactory;
+use Zend\Expressive\AppFactory;
+use Zend\Expressive\Router\Aura as AuraBridge;
+
+$auraRouter = (new RouterFactory())->newInstance();
+$auraRouter->setSecure(true);
```
Comment (bakura10) [inline doc/book/router.aura.md]: Are you sure about this? It does not seem to work here. The setSecure is defined in the Aura\Route, not the Router.

File: doc/book/router.aura.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+$auraRouter->addValues([
```
Comment (bakura10) [inline doc/book/router.aura.md]: Same.

Comment (weierophinney) [inline doc/book/router.aura.md]: Pretty sure on both: https://github.com/auraphp/Aura.Router#default-route-specifications details using these on the `$router` instance to set defaults.

File: doc/book/router.aura.md (hunk 1 above)
Comment (harikt) [inline doc/book/router.aura.md]: Aura.Router have `setSecure` method . See https://github.com/auraphp/Aura.Router#extended-route-specification

File: doc/book/router.aura.md (hunk 2 above)
Comment (harikt) [inline doc/book/router.aura.md]: aah @weierophinney have already commented on this. :+1:  .

**Pull request merged**